From e8d32c2cc5daab7e22f6190a02c88981aa14df7e Mon Sep 17 00:00:00 2001
From: Li Jun <jun.li@freescale.com>
Date: Fri, 3 Jul 2015 14:12:25 +0800
Subject: [PATCH] MLK-11123-28 ARM: dts: disable otg features for imx6sx-sdb
 board

Disable all otg features for otg1 port of imx6sx-sdb board, then
the otg port can only do role swtich by ID pin state.

Signed-off-by: Li Jun <jun.li@freescale.com>
---
 arch/arm/boot/dts/imx6sx-sdb.dts | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm/boot/dts/imx6sx-sdb.dts b/arch/arm/boot/dts/imx6sx-sdb.dts
index 20fd81b..10b594f 100644
--- a/arch/arm/boot/dts/imx6sx-sdb.dts
+++ b/arch/arm/boot/dts/imx6sx-sdb.dts
@@ -444,6 +444,9 @@
 	vbus-supply = <&reg_usb_otg1_vbus>;
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_usb_otg1_id>;
+	srp-disable;
+	hnp-disable;
+	adp-disable;
 	status = "okay";
 };
 
-- 
1.8.0

